src/cdk/text-field/autofill.spec.ts

```diff
@@ -192,14 +192,16 @@ describe('cdkAutofill', () => {
   `
 })
 class Inputs {
-  @ViewChild('input1') input1: ElementRef;
-  @ViewChild('input2') input2: ElementRef;
-  @ViewChild('input3') input3: ElementRef;
+  // Cast to `any` so we can stub out some methods in the tests.
```

Comment:
Is it possible to cast as an input element? `ElementRef<HTMLInputElement>`

Reply:
That's what I had initially, but TS complains because we stub out `addEventListener` in a few of the tests.